Gemfile

```diff
@@ -43,7 +43,7 @@ gem "tzinfo-data", platforms: %i[ mingw mswin x64_mingw jruby ]
 gem "bootsnap", require: false
 
 # Use Sass to process CSS
```

Comment:
1. use correct branch name. It should specify correct functionaliy
2. Why PR is included .DS_Store?
3. Why there are multiple images included? You're not using all of those